iree/compiler/Dialect/Flow/Transforms/DispatchLinalgOnTensors.cpp

```diff
@@ -144,6 +144,34 @@ static SmallVector<Value, 4> convertToWorkload(OpBuilder &b, Location loc,
   return workload;
 }
 
+/// Returns the fusion groups for the given `op`.
+static SmallVector<int64_t, 1> getFunsionGroups(Operation *op) {
```

Comment:
Typo : `getFunsionGroups` -> `getFusionGroups`

Reply:
THis doesnt seem to be resolved.

Reply:
It's done in ba4ec3c.